crypt4gh/keys/__init__.py

```diff
@@ -155,6 +157,29 @@ def run(argv=sys.argv[1:]):
 
     pubkey = os.path.expanduser(args['--pk'])
     seckey = os.path.expanduser(args['--sk'])
+    comment = args['-C'].encode() if args['-C'] else None
+
+    if args['--reencrypt']:
+        seckeypath = os.path.expanduser(seckey)
+        if not os.path.exists(seckeypath):
+            raise ValueError('Secret key not found')
+        old_passphrase = os.getenv('C4GH_PASSPHRASE')
+        cb = lambda: old_passphrase if old_passphrase else partial(getpass, prompt=f'Passphrase for {seckey}: ')
```

Comment:
What if it wasn't locked, or the `old_passphrase` is empty ? For the second case, we could use `if old_passphrase is not None` instead of allowing the empty string.